install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -3470,6 +3462,13 @@ clustermesh:
     #     cert: ""
     #     key: ""
     #     caCert: ""
+  globalNamespace:
+    # -- Default behavior of namespaces in Clustermesh.
+    # If set to true, all namespaces are considered global unless
+    # annotated otherwise. If set to false, all namespaces are considered
+    # local unless annotated otherwise.
+    # Defaults to true.
+    default: true
```

Comment:
Do we really want to enabled this by default? It seems like it is a potentially breaking change?

I also would think a bit if there's a better name for this configuration flag. Maybe something like `globalNamespace.annotationDefaultValue` or `annotationFallbackValue`

Reply:
> Do you want me to remove the commit altogether or do we want to add this as a comment for now?

I'd probably just remove it, comments would still be user visible, and potentially trick them into enabling a half-baked feature in case not all pieces make into v1.19.